### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by theneonlobster**

### CODE REVIEW

**Review by blaquehat — COMMENTED** [review]
@theneonlobster, the changes work. I tried them and only came across RAR with the following:

```
> drupal:sync:db
[Acquia\Blt\Robo\Tasks\DrushTask] Running /app/vendor/bin/drush cache-clear drush --uri=rar --no-interaction --ansi &&
 /app/vendor/bin/drush sql-sync @rar.dev @self --target-dump=/tmp/tmp.target.sql.gz --structure-tables-key=lightweight --create-db --uri=rar --no-interaction --ansi &&
 /app/vendor/bin/drush cr --uri=rar --no-interaction --ansi &&
 /app/vendor/bin/drush sql-sanitize --uri=rar --no-interaction --ansi in /app/docroot
 [success] 'drush' cache was cleared.
You will destroy data in drupal_rar and replace with data from uwmeddev.ssh.prod.acquia-sites.com/uwmeddevdb348575.
 [notice] Starting to create database on target.

In Process.php line 235:

  The command "/app/vendor/drush/drush/drush sql-create --create-db --target-dump=/tmp/tmp.target.sql.gz --ansi --no-interaction --structure-tables-key=lightweight --strict=0 --uri=rar --root=/app/docroot" failed.

  Exit Code: 1(General error)

  Working directory:

  Output:
  ================
  Creating database drupal_rar. Any existing database will be dropped!

  Error Output:
  ================

  In SqlCommands.php line 104:

    Unable to create database. Rerun with --debug to see any error message.

  sql:create [--db-su DB-SU] [--db-su-pw DB-SU-PW] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-d|--debug] [-y|--yes] [--no] [--remote-host REMOTE-HOST] [--remote-user REMOTE-USER] [-r|-
  -root ROOT] [-l|--uri URI] [--simulate] [--pipe] [-D|--define DEFINE] [--database [DATABASE]] [--target [TARGET]] [--db-url DB-URL] [--druplicon] [--notify [NOTIFY]] [--xh-link XH-LINK] [--] <command>

sql:sync [--no-dump] [--no-sync] [--runner RUNNER] [--create-db] [--db-su DB-SU] [--db-su-pw DB-SU-PW] [--target-dump TARGET-DUMP] [--source-dump [SOURCE-DUMP]] [--extra-dump] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-d|--debug] [-y|--yes] [--no] [--remote-host REMOTE-HOST] [--remote-user REMOTE-USER] [-r|--root ROOT] [-l|--uri URI] [--simulate] [--pipe] [-D|--define DEFINE] [--skip-tables-key SKIP-TABLES-KEY] [--structure-tables-key STRUCTURE-TABLES-KEY] [--tables-key TABLES-KEY] [--skip-tables-list SKIP-TABLES-LIST] [--structure-tables-list STRUCTURE-TABLES-LIST] [--tables-list TABLES-LIST] [--druplicon] [--notify] [--xh-link XH-LINK] [--] <command> <source> <target>

[Acquia\Blt\Robo\Tasks\DrushTask]  Exit code 1  Time 6.336s
[error]  Command `drupal:sync:db ` exited with code 1. This most likely indicates a problem with your configuration, and is not a BLT bug.
For troubleshooting guidance and support, see https://docs.acquia.com/blt/support/
[error]  Command `drupal:sync:default:site ` exited with code 1. This most likely indicates a problem with your configuration, and is not a BLT bug.
For troubleshooting guidance and support, see https://docs.acquia.com/blt/support/
```

Another suggestion would be to make RAR's local.settings.php file the same as Stevie's.

**Review by blaquehat — APPROVED** [review]
After going over what I found with @theneonlobster, It's ready to proceed.

File: docroot/sites/stevie/blt.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ project:
   human_name: stevie
   local:
     protocol: http
-    hostname: stevie.uwmed.local
+    hostname: stevie.lndo.site
```
[the remark this answers is not in the document]
Comment (blaquehat) [reply docroot/sites/stevie/blt.yml:11]: @sodacrackers, lndo.site is Lando's default. The ssl certificate uses that unless you override it.

File: docroot/sites/rar/blt.yml [hunk 2]
```diff
@@ -1,25 +1,18 @@
...
     remote: rar.dev
-    local: rar.local
+    local: self
+cm.core.dirs.sync.path: '${cm.core.path}/rar'
+project:
+  machine_name: rar
+  human_name: rar
+  local:
+    protocol: http
+    hostname: rar.lndo.site
```
[the remark this answers is not in the document]
Comment (blaquehat) [reply docroot/sites/rar/blt.yml:11]: @sodacrackers, lndo.site is Lando's default. The ssl certificate uses that unless you override it.

File: docroot/sites/huddle/settings/default.local.settings.php [hunk 3]
```diff
@@ -15,23 +15,37 @@
...
+ */
+$info = json_decode(getenv('LANDO_INFO'), TRUE);
+
+/**
+ * NOTE: If $info is not empty then use Lando related settings.
+ */
+if (!empty($info)) {
+  // NOTE: Database configuration for Lando setups - adds in Port and Host.
+  $databases['default']['default']['port'] = $info['database']['internal_connection']['port'];
+  $databases['default']['default']['host'] = 'huddle_db';
```
[the remark this answers is not in the document]
Comment (theneonlobster) [reply docroot/sites/huddle/settings/default.local.settings.php:38]: This is using Lando's defaults, per the previously-committed docroot/sites/stevie/settings/default.local.settings.php

File: docroot/sites/chew/settings/default.local.settings.php [hunk 4]
```diff
@@ -12,27 +12,40 @@
...
+ */
+$info = json_decode(getenv('LANDO_INFO'), TRUE);
+
+/**
+ * NOTE: If $info is not empty then use Lando related settings.
+ */
+if (!empty($info)) {
+  // NOTE: Database configuration for Lando setups - adds in Port and Host.
+  $databases['default']['default']['port'] = $info['database']['internal_connection']['port'];
+  $databases['default']['default']['host'] = 'chew_db';
```
[the remark this answers is not in the document]
Comment (theneonlobster) [reply docroot/sites/chew/settings/default.local.settings.php:38]: This is using Lando's defaults, per the previously-committed docroot/sites/stevie/settings/default.local.settings.php

File: docroot/sites/chew/settings/default.local.settings.php [hunk 5]
```diff
@@ -12,27 +12,40 @@
...
+
+  $settings['hash_salt'] = md5(getenv('LANDO_HOST_IP'));
+
+  $base_url = (isset($_SERVER['HTTPS']) ? 'https' : 'http') . '://' . $_SERVER['HTTP_HOST'];
+} else {
+  // NOTE: Database configuration for non-Lando setups - adds in Port and Host.
+  $databases['default']['default']['port'] = '3306';
```
[the remark this answers is not in the document]
Comment (theneonlobster) [reply docroot/sites/chew/settings/default.local.settings.php:45]: This is using Lando's defaults, per the previously-committed docroot/sites/stevie/settings/default.local.settings.php

**Comment by theneonlobster** [comment]
Rebasing and merging! (I enabled that on the repo.)

**Pull request merged**